tests/core/variant/test_variant.h

```diff
@@ -1810,7 +1810,7 @@ TEST_CASE("[Variant] Writer key sorting") {
 	String d_str;
 	VariantWriter::write_to_string(d, d_str);
 
-	CHECK_EQ(d_str, "{\n\"A\": 1,\n&\"B\": 2,\n&\"C\": 3,\n\"D\": 4\n}");
+	CHECK_EQ(d_str, "{\n\"D\": 4,\n&\"B\": 2,\n\"A\": 1,\n&\"C\": 3\n}");
 }
```

Comment:
This whole test should be removed as it tests for the sorting of the keys, or should be renamed to make it clear that it tests it keeps insertion order